20010-07-01		Magnus Granberg <zorry@gentoo.org>	Toth Attila <atoth@atoth.sote.hu>

	* src/config/Linux_All.mk	Change LD to CC

--- a/src/config/Linux_All.mk	2009-08-29 12:11:48.000000000 +0200
+++ b/src/config/Linux_All.mk	2009-08-29 12:13:13.000000000 +0200
@@ -45,7 +45,7 @@
 OS_CFLAGS = -DXP_UNIX -DSVR4 -DSYSV -D_BSD_SOURCE -DPOSIX_SOURCE -DHAVE_LOCALTIME_R
 
 RANLIB = echo
-MKSHLIB = $(LD) -shared $(XMKSHLIBOPTS) -soname $(notdir $@)
+MKSHLIB = $(CC) -shared $(XMKSHLIBOPTS) -Wl,-soname -Wl,$(notdir $@)
 
 #.c.o:
 #      $(CC) -c -MD $*.d $(CFLAGS) $<
